### PR TITLE
Completed ticket 12 delete

### PR DESCRIPTION
**Pull request by claire-farrow**
- completed delete on api/comments

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -261,5 +246,14 @@ describe("/api/users", () => {
   });
 });
 
+describe('/api/comments', () => {
+  describe("DELETE", () => {
+    test("Status 204 - Delete comment entry by comment_id", () => {
+      const id = 2;
+      return request(app).delete(`/api/comments/${id}`).expect(204);
+    })
```
Comment (scar1377) [inline __tests__/app.test.js:254]: Think of a way to check the comment has actually been deleted

File: controllers/comments.controllers.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const { removeCommentById } = require("../models/comments.models");
+
+exports.deleteCommentById = (req, res, next) => {
+    const id = req.params.comment_id;
+    removeCommentById(id)
+    .then((body) => {
+        res.status(204).send({body});
```
Comment (scar1377) [inline controllers/comments.controllers.js:7]: DELETE method doesn't return anything. Can leave .send() empty or use .end()instead

**Review by scar1377 — CHANGES_REQUESTED** [review]
missing test

**Review by Angaca — APPROVED** [review]
Good to go!

**Pull request merged**